data-es5.js

```diff
@@ -1170,6 +1225,8 @@ exports.tests = [
     return typeof Array.prototype.every == 'function';
   },
   res: {
+    es5shim: assign({ val: true }, sparseNote),
```

Comment:
I realized that afterwards - but this way, the HTML isn't arbitrarily encoded in only one of them (like in es6), and I don't have to use strings to add the note to a new one later.